render/board.go

```diff
@@ -142,6 +142,11 @@ func (b *Board) addHazard(p *engine.Point) {
 }
 
 func getDirection(p engine.Point, nP engine.Point) snakeDirection {
+	// handle when the points haven't changed (a common case) by defaulting to up
```

Comment:
this was really spammy in the logs because it is common to have these two points be the same (nothing is actually wrong)

Reply:
Makes sense. For reasons far too ridiculous to explain... can you set the default to face right rather than up? 

It's kind of a thing in the Battlesnake community :)
<img width="381" alt="Screen Shot 2022-02-07 at 7 41 23 PM" src="https://user-images.githubusercontent.com/1531419/152913853-590e7914-a197-47a1-8863-ec8c013253eb.png">